Recommend unique model-run names (#152)
The-Strategy-Unit/nhp_project_information: user_guide/naming_scenarios.qmd

---
title: "How to name model scenarios"
order: 3
---

# Naming your model runs

Name your model runs carefully. A good name will help you and the model roll-out team understand what's in your model, when it was created and when you ran it.

> In particular, we recommend that you give each model run a unique name, which will help prevent errors in communication or processing.

Reviewer: ```suggestion
In particular, each model run must have a unique name, which will help prevent errors in communication or processing.

```

It's important to realise that at the outputs stage you will have access to the name of the model and the date and time of when it was run, but not the date and time when you created it. 

> It's possible to select an existing model run from the inputs app and re-run it with tweaked parameters. This could result in model runs with the same name but different parameters, although they'll be differentiated in the outputs app by the date and time they were run. We suggest you create a new name if you've adjusted the parameters, which will help differentiate these runs.

Reviewer: I would be stronger with the language here. Also adding a sentence for _how_ to rename a scenario (as per @jentwood

```suggestion
It's possible to select an existing model run from the inputs app and re-run it with tweaked parameters. This could result in model runs with the same name but different parameters, although they'll be differentiated in the outputs app by the date and time they were run. 

If you have already run a scenario and wish to edit the parameters after this, you must change the name of the scenario. You can do this by selecting "Create new from existing" on the inputs app.

```
